web/js/function-highlight-addon.js

```diff
@@ -0,0 +1,176 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
That file is a modified copy of a CodeMirror file, which is MIT licensed -- so my interpretation is that we can relicense it as long as we keep the copyright?

Is the third_party directory designed for files which CodeWorld is modifying, or just for straight use of dependencies?